Debian/bin/lib/Cyrus/Docker/Command/test.pm

```diff
@@ -0,0 +1,59 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::test;
+use Cyrus::Docker -command;
+
+use Path::Tiny;
+use Process::Status;
+
+sub execute ($self, $opt, $args) {
+  unless (-e '/run/rsyslogd.pid') {
+    system('/usr/sbin/rsyslogd');
+    Process::Status->assert_ok('starting rsyslog');
+  }
+
+  my $root = "/srv/cyrus-imapd/cassandane";
+  chdir $root or die "can't chdir to $root: $!";
+
+  unless (-e "cassandane.ini") {
+    system(qw(cp -af cassandane.ini.dockertests cassandane.ini));
+    Process::Status->assert_ok('copying cassandane.ini.dockertests to cassandane.ini');
+
+    system(qw(chown cyrus:mail cassandane.ini));
+    Process::Status->assert_ok('chowning cassandane.ini');
+  }
+
+  # XXX This is transitional, while we haven't updated cyrus-imap.git to
+  # eliminate the .git in path names that existed prior to recent commits.
+  {
+    my @lines = path('cassandane.ini')->lines;
+    s{/srv/[-A-Za-z]+\K.git}{}g for @lines;
+    path('cassandane.ini')->spew(@lines);
+  }
+
+  # XXX This is transitional, while we haven't updated cyrus-imap.git to
+  # eliminate the .git in path names that existed prior to recent commits.
+  {
+    my @lines = path('cassandane.ini')->lines;
+    s{/srv/[-A-Za-z]+\K.git}{}g for @lines;
+    path('cassandane.ini')->spew(@lines);
+  }
+
+  system(qw(make -j 12));
+  Process::Status->assert_ok('Cassandane make');
+
+  my @opts = $ENV{CASSANDANEOPTS} ? $ENV{CASSANDANEOPTS} : ();
+
+  system(
+    qw(
+      setpriv --reuid=cyrus --regid=mail
+              --clear-groups --inh-caps=-all
+              ./testrunner.pl -f pretty -j 12
+    ),
+    @opts,
```

Comment:
I modified mine to pass in @$args so I can say `cyd test Cyrus::JMAPSieve.sieve_test` for example, or add `-vvv`, etc etc

Reply:
Yeah, as it stands this code is mehhh and I want to fix it in next round of work.

Reply:
This is a quick easy win I think you'd find useful!

Reply:
I want to do it soon, but I want to get the design right.  Think about it this way:
* `checkout` and `build` should take options (check out branch X, build with option Y)
* `test` should take options (only run these six tests)
* `smoke` is a way to run the tests in a non-interactive container
* `smoke` should be able to combine options for the above usefully

So I don't want to set up the opts/args for these commands yet.  I want to have a strategy for how smoke composes them, then do all of it.  That's the next step.

Reply:
Okay.